### DISCUSSION TITLE
How to change the vertical alignment

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [marp-team/marp#458] [Q&A] [answered]
What css should I use to use to position the content towards the top of the slides?
I've tried the following but no luck:

```
section {
  position: absolute;
  top: 0px;
}
```

And the following only changes the headers:
```
h2{
  position: absolute;
  top: 50px;
}
```

**Comment by yhatt** [comment, marked as answer]
Refer to https://github.com/marp-team/marp-core/issues/78#issuecomment-474416409.

`position: absolute` is suitable to fix the electively element to the top. To disable centering for every contents, try this:

```css
section {
  justify-content: flex-start;
}
```

This tweak works with `default` theme and `uncover` theme.

**Reply by nadiamoe** [reply]
@yhatt This does not seem to work anymore in marpit v4, is there an alternative way of doing it in v4? Thanks!

**Reply by yhatt** [reply]
@roobre Marp Core v4 has been changed the display style of slide containers from Flexbox into the block element (#533). `justify-content` is only for Flexbox so that is no longer working. Insted, please use [`align-content`](https://developer.mozilla.org/docs/Web/CSS/align-content) property.

```css
section {
  align-content: start;
}
```

**Reply by nadiamoe** [reply]
That did the trick, thank you! ❤️

**Comment by SoumayaMauthoorMOJ** [comment]
Perfect thank you :-)
